webapp/src/ts/modules/tasks/tasks.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="filters">
+  <!--div class="inner">
+    <mm-navigation></mm-navigation>
+  </div-->
+</div>
+
+<div class="inner">
+  <div class="inbox page">
+
+    <div id="tasks-list" class="col-sm-4 inbox-items left-pane">
+      <div class="selection-count" *ngIf="selectMode">
+        <span *ngIf="selectedTasks.length === 1" translate>select.mode.count.singular</span>
+        <span *ngIf="selectedTasks.length !== 1" translate [translateParams]="{ number: selectedTasks.length || 0 }">select.mode.count.plural</span>
+      </div>
+      <ul class="unfiltered" *ngIf="(!loading || appending) && !filtered">
```

Comment:
Is the `unfiltered` class necessary here? We don't have task filtering yet.  Please use the same tasks and conditions used in this file: https://github.com/medic/cht-core/blob/master/webapp/src/templates/directives/tasks_list.html